app/services/bgs/awards_service.rb

```diff
@@ -11,7 +11,9 @@ class AwardsService < BaseService
     # @return [Hash]
     #
     def get_awards
-      @service.awards.find_award_by_participant_id(@user.participant_id, @user.ssn)
+      response = @service.awards.find_award_by_participant_id(@user.participant_id, @user.ssn)
+      response = @service.awards.find_award_by_ssn(@user.ssn) if response.nil?
+      response
```

Comment:
```suggestion
      @service.awards.find_award_by_participant_id(@user.participant_id, @user.ssn) ||
        @service.awards.find_award_by_ssn(@user.ssn)
```

What do you think of this?